CardanoSharp.Koios.Client/CardanoSharp.Koios.Client.csproj

```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <TargetFramework>netstandard2.1</TargetFramework>
         <Nullable>enable</Nullable>
-        <Version>1.0.9</Version>
+        <Version>1.0.10</Version>
```

Comment:
Couldn't avoid this version drift due to nuget only supporting first three {major.minor.patch} parts of semantic versioning and we had to apply a revision update for the last version (1.0.8). 

We have to wait until a minor or major version bump from Koios to re-sync the versioning.